src/bun.js/bindings/JSMockFunction.cpp

```diff
@@ -295,6 +345,15 @@ class JSMockFunction : public JSC::InternalFunction {
         }
         return val;
     }
+    JSArray* getInvocationCallOrder() const
+    {
+        JSArray* val = invocationCallOrder.get();
```

Comment:
we could avoid this by saying `if (invocationCallOrder)` which will check if the WriteBarrier has never been written to

but this is also fine